JavaAssignments/src/mayur/FahrenheitToCelsius.java

```diff
@@ -0,0 +1,25 @@
+package mayur;
+
+public class FahrenheitToCelsius 
+{
+	static int Celsius;
+
+	public static void main(String[] args) 
+	{
+		FahrenheitToCelsius f = new FahrenheitToCelsius();
+		float fahrenheit = (float) (Double.parseDouble(args[0]));
```

Comment:
why you are converting double to float. 
double fahrenheit = Double.parseDouble(args[0]);